scripts/type-generation/webdriver-generate-typings.js

```diff
@@ -51,13 +51,15 @@ for (const [protocolName, definition] of Object.entries(PROTOCOLS)) {
         }
     }
 
-    lines.push('    }\n')
+    lines.push('}')
 }
 
 const template = fs.readFileSync(TEMPLATE_PATH, 'utf8')
-const outputFile = path.join(__dirname, '..', '..', 'packages', 'webdriver', 'webdriver.d.ts')
-const generatedTypings = EDIT_WARNING + template.replace('// ... insert here ...', lines.join('\n'))
-fs.writeFileSync(outputFile, generatedTypings, { encoding: 'utf-8' })
+const outputFile = path.join(__dirname, '..', '..', 'packages', 'webdriver', 'build', 'types.d.ts')
+const generatedTypings = template.replace('// ... insert here ...', lines.join('\n'))
+
+const origTypings = fs.readFileSync(outputFile, 'utf8')
```

Comment:
```suggestion
const origTypings = fs.readFileSync(outputFile, { encoding: 'utf8' })
```

Reply:
If options is passed as a string it is automatically used to specify encoding (see https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_readfile_path_options_callback)